server/jwt.go

```diff
@@ -132,7 +132,7 @@ func validateTrustedOperators(o *Options) error {
 		if o.TrustedKeys == nil {
```

Comment:
Please change this to use o.DidSign(o) as implements this logic without performing any of these checks that duplicate something that JWT can do correctly. If the JWT deserialized properly, and DidSign returns true, the JWT was signed by one of the listed operator keys.

Reply:
I think this is to support old configurations...
For kicks I added the following check when we iterate over trusted operators.
Only last one triggers

		vr := jwt.ValidationResults{}
		opc.Validate(&vr)
		if !vr.IsEmpty() {
			return fmt.Errorf("operator not valid %v", vr)
		}
		if !opc.DidSign(opc) {
			return fmt.Errorf("operator not self signed DidSign")
		}
		if !opc.IsSelfSigned() {
			return fmt.Errorf("operator not self self signed IsSelfSigned")
		}